notify/slack/slack.go

```diff
@@ -210,16 +210,66 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 	if err != nil {
 		return true, notify.RedactURL(err)
 	}
-	defer notify.Drain(resp)
+	defer resp.Body.Close()
```

Comment:
I'm not 100% familiar with `notify.Drain`, but why would we stop using it here? The call is deferred, so we can still read `resp.Body` from the function call on Line 226.

Reply:
Thank you!